.github/workflows/ coderabbit.yml

```
@@ -0,0 +1,14 @@
+name: Code Review with CodeRabbit
+
+on: [push, pull_request]
+
+jobs:
+  review:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
```

actionlint

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)